### DISCUSSION TITLE
Default theme style usage (Office hours: Oct 1, 2020)

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#804] [Office hours] [answered]
Help Needed: Want to confirm if I am using the paste components in default style correctly and whether I am complying with all the UDL expectations. 

Background - In an effort to create a cohesive experience, we are visually updating the console login pages. The page are still in PHP so need to create a stopgap solution to at least visually align the pages until we do a full redesign.
Stage of design: Exploring
Links to mocks or prototype: https://share.goabstract.com/11001318-74bf-4b83-8017-0d761d53ba9a

**Comment by serifluous** [comment, marked as answer]
Hey @pguptaDS ! Can you give a little more clarity on what questions you're hoping to get answered?

**Reply by pguptaDS** [reply]
Just added some in the description above. @serifluous

**Reply by serifluous** [reply]
Thank you! I left some comments and questions on Abstract. Overall, the use of the default theme is really great.

It'll be worth talking through the error cases in office hours, along with any of the other comments you want to discuss.

**Reply by richbachman** [reply]
This is looking really good. I just left a couple small comments in Abstract.
